filesystem/usr/local/share/landoop/sample-data/lenses-processor.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+LENSES_PORT=${LENSES_PORT:-9991}
+USER=${USER:-admin}
+PASSWORD=${PASSWORD:-admin}
+
+# Wait for Lenses to get up if needed and see the topic
+for ((i=0;i<30;i++)); do
+   sleep 5
+   lenses-cli --timeout 3s --user "$USER" --pass "$PASSWORD" --host http://127.0.0.1:$LENSES_PORT topics \
+   | grep -sq "sea_vessel_position_reports" && { sleep 2; break; };
+done
```

Comment:
You don't make sure the topic is detected as AVRO/AVRO here.

Reply:
I'll add a condition for checking whenever the two columns are the same and equal to AVRO and then proceed with processor's creation